css/src/components/table.css

```diff
@@ -0,0 +1,213 @@
+.yoast table.yoast-table {
+	font-size: var(--font-size-default);
+	color: var(--color-default);
+	line-height: 1.2;
+	border-bottom: var(--border-default);
+}
+
+.yoast .yoast-table tfoot th {
+	border-bottom: none;
+}
+
+.yoast .yoast-table th {
+	border-bottom: var(--border-default);
+}
+
+.yoast .yoast-table th.check-column {
+	padding: 19px 0 0 3px;
```

Comment:
I guess this is for Premium so I can't test it on this PR. Will test on the Premium PR.